src/extlibs/enlsvg/LineOfSightScanner.cpp

```diff
@@ -0,0 +1,556 @@
+#include "extlibs/enlsvg/LineOfSightScanner.h"
```

Comment:
this file is overwritten with "line_of_sight_scanner.cpp", so it shouldn't exist